package.json

```
@@ -80,6 +80,32 @@
     "browser": "./dist/extension.browser.js",
     "l10n": "./l10n",
     "contributes": {
+        "problemMatchers":
+        [
+            {
+                "name": "python",
+                "owner": "python",
+                "source": "python",
+                "fileLocation": "autoDetect",
+                "pattern": [
+                    {
+                        "regexp": "^.*File \\\"([^\\\"]|.*)\\\", line (\\d+).*",
+                        "file": 1,
+                        "line": 2,
+                        "severity": 4,
+                        "message": 5
+                    },
+                    {
+                        "regexp": "^\\s*(.*)\\s*$"
+
+                    },
+                    {
+                        "regexp": "\\s*(.*Error.*)$",
```

This could also have been pinned to the beginning of the line:
```suggestion
                        "regexp": "^\\s*(.*Error.*)$",
```

---

great point, thank u for pointing this out